tests/comments.imp

```diff
@@ -0,0 +1,9 @@
+# PROGRAM
+main # MAIN SUBROUTINE
+    # BEGIN BLOCK
+    begin
+        # PRINT A HELLO WORLD... IN LATIN
```

Comment:
Why are the comments yelling? :cry:

Reply:
They try to be part of the code, but fail miserably.